src/test/java/lambda/part1/exercise/Lambdas01Exercise.java

```diff
@@ -21,6 +22,7 @@ public void sortPersonsByAge() {
                 new Person("name 2", "lastName 1", 30)
         };
 
+        Arrays.sort(persons, (a, b) -> b.getFirstName().compareTo(a.getFirstName()));

```

Comment:
`Lambdas01Exercise` - use anonymous classes, not lambdas.
`sortPersonsByAge` - compare "age".